plugins/polio/api.py

```diff
@@ -185,6 +187,8 @@ def filter_queryset(self, queryset):
             campaigns = campaigns.filter(is_preventive=False).filter(is_test=False)
         if campaign_groups:
             campaigns = campaigns.filter(grouped_campaigns__in=campaign_groups.split(","))
+        if org_unit_groups:
+            campaigns = campaigns.filter(initial_org_unit__groups__in=org_unit_groups.split(","))
```

Comment:
idem filter on `country` not `initial_org_unit`

Reply:
fixed